output_template/python/lmnet/utils/demo.py

```diff
@@ -49,19 +49,18 @@ def add_fps(orig, fps):
     return orig
 
 
-def check_range(upper, lower, checked_val):
-    if upper < checked_val:
-        checked_val = upper
-    elif lower > checked_val:
-        checked_val = lower
-    return checked_val
+def clamp(lower, upper, target):
```

Comment:
`clamp(target, lower, upper)` might be more straightforward.
- https://cpprefjp.github.io/reference/algorithm/clamp.html
- https://numpy.org/doc/1.18/reference/generated/numpy.clip.html